Scripts/main.js

```diff
@@ -192,11 +192,12 @@ function cliPath() {
 }
 
 function getSetting(section, key) {
-  try {
```

Comment:
The docs don't mention [open](https://docs.nova.app/api-reference/file-system/#openpath-mode-encoding) throwing an error, but it doesn't hurt to leave this exception handling here.